app/utils.py

```diff
@@ -5,6 +5,7 @@
 
 
 def link(rel, href):
+    """Generates a link dict from a rel, href pair."""
```

Comment:
Should be "Generate a link ...", according to https://www.python.org/dev/peps/pep-0257/#one-line-docstrings